apps/dotcom/client/e2e/tests/homepage.spec.ts

```diff
@@ -16,10 +16,40 @@ test('can toggle sidebar', async ({ editor, sidebar }) => {
 	await expect(sidebar.sidebarLogo).toBeVisible()
 })
 
-test('can create new document', async ({ editor, sidebar }) => {
+test('can create new file', async ({ editor, sidebar }) => {
 	await editor.ensureSidebarOpen()
 	const currentCount = await sidebar.getNumberOfFiles()
 	await sidebar.createNewDocument()
 	const newCount = await sidebar.getNumberOfFiles()
 	expect(newCount).toBe(currentCount + 1)
 })
+
+test.fixme('can can double click file name to rename it', async () => {
+	// ...
+})
+
+// Preferences
+
+test.fixme('can toggle dark mode', async () => {
+	// ...
+})
+
+// File menu in sidebar
+
+test.fixme('can duplicate a file', async () => {
+	// ...
+})
+
+test.fixme('can copy a file link from the file menu', async () => {
+	// ...
+})
+
+test.fixme('can delete a file', async () => {
```

Comment:
@steveruizok fwiw @Taha-Hassan-Git is looking into this in this PR https://github.com/tldraw/tldraw/pull/4827

cc @Taha-Hassan-Git just so you know about these new stubs in relation to your change